bit_manipulation/decimal_to_binary.py

```diff
@@ -0,0 +1,43 @@
+def decimal_to_binary(number: int) -> str:
```

Comment:
I don't see any decimal number in the tests. Please provide tests for numbers like `13.321312312`